app/src/main/res/xml/appfilter.xml

```diff
@@ -311,7 +320,7 @@
     <item component="ComponentInfo{nl.asnbank.asnbankieren/com.service2media.m2active.client.android.M2ActiveClient}" drawable="bankieren" />
 
     <!-- Banque -->
-    <item component="ComponentInfo{com.caisseepargne.android.mobilebanking/com.caisseepargne.android.mobilebanking.activities.ADashBoard}" drawable="banque" />
+    <item component="ComponentInfo{com.caisseepargne.android.mobilebanking/com.caisseepargne.android.mobilebanking.features.main.MainActivity}" drawable="banque" />
```

Comment:
Generally I keep both so people with old apps still get the icons.

Reply:
I get the idea. Fixed it.